management-app/src/main/java/com/sdevelopment/skt/management/controller/ProductController.java

```diff
@@ -0,0 +1,56 @@
+package com.sdevelopment.skt.management.controller;
+
+import com.sdevelopment.skt.common.domain.Product;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.validation.Valid;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+public class ProductController {
+
+    @RequestMapping("/create-product")
+    public String createProduct(ModelMap model) {
```

Comment:
by default, `@RequestMapping` sets the endpoint as a `GET` method.
if we are creating a new resource in our system, a `POST` method ought to be used.

Reply:
Yes. Saw that in a documentation or article.